### PR TITLE
Fix: (Rosetta-Code) Correct the Test suite and make expression clearer

### PR DESCRIPTION
**Pull request by obbap1**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Implement a function that takes a string of four digits as its argument, with ea
 | Example input | Example output |
 | --- | --- |
 | <code>solve24("4878");</code> | <code>(7-8/8)*4</code> |
-| <code>solve24("1234");</code> | <code>3*1*4*2</code> |
+| <code>solve24("1234");</code> | <code>3 * 1 * 4 * 2</code> |
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md]: @obbap1 I don't think this is necessary.  Also, if you are going to change one test `text`, you should really changed them all.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@obbap1 See my suggested comment.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md]: Got it.  In that case, let's change the table entry to be:
```md
| <code>solve24("1234");</code> | <code>3*1*4*2</code> |
```
and remove the spaces in your change.

Comment (obbap1) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md]: With that, it looks like this
image

You can try it directly as well

Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md]: It has to do with the way that markdown is display using asterisks.  

Try the following and let me know.
```
| <code>solve24("1234");</code> | <code>3&ast;1&ast;4&ast;2</code> |
```

Comment (obbap1) [reply curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md]: Okay, it works fine now.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
I don’t understand why we need the latter hangs at all. @obbap1 what am I missing? 🤔

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

@thecodingaviator it all seems fine to me, which bit didn't you think was necessary?

**Comment by thecodingaviator** [comment]
@ojeytonwilliams https://github.com/freeCodeCamp/freeCodeCamp/pull/38102/files#diff-53c7f2b7212690d0d226bc46cf83086dR47

**Comment by obbap1** [comment]
Hello @thecodingaviator, The change was necessary because the old syntax wasn't producing the expected result.

https://github.com/freeCodeCamp/freeCodeCamp/pull/38102#discussion_r370213632

**Comment by ojeytonwilliams** [comment]
The `&ast;`s are necessary to render stars in Markdown.  Well, you could use `\*`, which is a bit terser, but either works.  Is that what you were referring to?

**Review by thecodingaviator — APPROVED** [review]

**Comment by thecodingaviator** [comment]
Thanks for contributing to the freeCodeCamp repository. Congratulations on your first PR! 🎊 

Look forward to more contributions from you! ✈️

**Pull request merged**